### DISCUSSION TITLE
Is Deno support planned?

### DISCUSSION THREAD

**Discussion by NetOpWibby** [zed-industries/zed#6607] [Feature Requests]
The squigglies are killing me

<img width="2113" alt="SS 2023-10-25 at 6 08 47 PM" src="https://github.com/zed-industries/community/assets/1288356/b1dd9771-2ec7-4f6b-a803-83593a78016b">

**Comment by bhb603** [comment]
Looks like Deno support was release in [v0.121.5](https://github.com/zed-industries/zed/releases/tag/v0.121.5) 🎉 

It works well, however I would like to propose the ability to enable it per-project, e.g. in "Folder-specific settings".

Right now it seems you need to enable it globally:
```jsonc
// ~/.config/zed/settings.json
{
    "deno": {
      "enable": true
   }
}
```

This is a problem when you have some Deno projects and some non-Deno (Nodejs+Typescript) projects... to switch between them I have to change global settings and restart Zed.

Any workarounds for this? Or can I vote to get it on the roadmap? 😄

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
